### PR TITLE
[DGPFET-10] The Software Developer conditionals

### PR DESCRIPTION
**Pull request by chudylobor**

### CODE REVIEW

File: submissions/godsonOgbuokiri/JavaScript/Conditionals/conditionals.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+let p = 123-7;
+for (let i = 1; i <= p; i++){
+	if(i % 3 == 0 && i % 5 == 0) {
+		document.write(" " + "Software Developer ");
```
Comment (Mayowa2020) [inline submissions/godsonOgbuokiri/JavaScript/Conditionals/conditionals.js:4]: Please add a break tag here, to make your work look more presentable.

File: submissions/godsonOgbuokiri/JavaScript/Conditionals/conditionals.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	}
+
+	 else if(i % 5 == 0) {
+		document.write(" " + "Developer");
```
Comment (Mayowa2020) [inline submissions/godsonOgbuokiri/JavaScript/Conditionals/conditionals.js:8]: Please add a break tag here, to make your work look more presentable.

File: submissions/godsonOgbuokiri/JavaScript/Conditionals/conditionals.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	}
+
+	else if(i % 3 == 0 ) {
+		document.write(" " + "Software");
```
Comment (Mayowa2020) [inline submissions/godsonOgbuokiri/JavaScript/Conditionals/conditionals.js:12]: Please add a break tag here, to make your work look more presentable.

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
@chudylobor, please effect the changes and revert. Thank!

**Review by Mayowa2020 — APPROVED** [review]
@chudylobor Well done. In the case of another time, kindly ensure a task is approved, before pushing in the next task. 

Approved...

**Comment by chudylobor** [comment]
Thanks
I have made all corrections

**Pull request merged**